new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -1117,6 +1118,50 @@ public static boolean createConfigMap(V1ConfigMap configMap) throws ApiException
     return true;
   }
 
+  /**
+   * Replace a Kubernetes Config Map.
```

Comment:
Can u explain bit more detail. What is the purpose of replacing a Config Map

Reply:
this is to support updates for CM ( kubectl edit cm ) I have added more description